build.gradle

```diff
@@ -159,6 +160,7 @@ allprojects {
         '-Werror', // Treat each compiler warning (esp. the ones coming from Checker Framework) as an error.
         '-Xlint:unchecked', // Warn of type-unsafe operations on generics.
       ]
+      options.release = 8
```

Comment:
This is about what Java APIs should be visible to our Java code... this is tricky b/c if you don't set that, then you can write Java 8 code calling Java 11 APIs (like `Path.of` or `InputStream#readAllBytes()`), which is bound to crash in runtime when running on Java 8-compatible JVM